src/quo2/components/empty_state/empty_state/styles.cljs

```diff
@@ -2,8 +2,11 @@
   (:require [quo2.foundations.colors :as colors]))
 
 (def container
-  {:padding     12
-   :align-items :center})
+  {:padding         12
+   :align-items     :center
+   :justify-content :center
+   :flex            1
+   :margin-bottom   44})
```

Comment:
hmm, this seems suspicious. Imo opinion we should not add margins to the container - better to use a container-style prop and merge it with this and set it on the page being used. often we can be fighting the page margins and the component margins otherwise 👍

Reply:
@J-Son89 it wasn't perfectly centered before (on all pages). This 44 margin bottom is the height of the `jump-to` container. I can ask Francesca to review it